--- orig/PKGBUILD	2025-02-28 18:01:37.640834355 -0600
+++ new/PKGBUILD	2025-05-31 01:40:59.517664100 -0500
@@ -7,10 +7,12 @@
 pkgname=dwm
 pkgver=6.5
 pkgrel=1
+pkgvariant=new
 pkgdesc="A dynamic window manager for X"
 url="https://dwm.suckless.org"
 arch=('i686' 'x86_64' 'arm' 'armv7h' 'armv6h' 'aarch64')
 license=('MIT')
+groups=('modified')
 options=(zipman)
 depends=('libx11' 'libxinerama' 'libxft' 'freetype2')
 install=dwm.install
@@ -22,19 +24,21 @@
             'SKIP')
 
 prepare() {
-  cd "$srcdir/$pkgname-$pkgver"
+  # rm -rf "$srcdir/$pkgname-$pkgver.$pkgvariant"
+  cp -rT "$srcdir/$pkgname-$pkgver" "$srcdir/$pkgname-$pkgver.$pkgvariant"
+  cd "$srcdir/$pkgname-$pkgver.$pkgvariant"
   if [[ -f "$srcdir/config.h" ]]; then
     cp -fv "$srcdir/config.h" config.h
   fi
 }
 
 build() {
-  cd "$srcdir/$pkgname-$pkgver"
+  cd "$srcdir/$pkgname-$pkgver.$pkgvariant"
   make X11INC=/usr/include/X11 X11LIB=/usr/lib/X11 FREETYPEINC=/usr/include/freetype2
 }
 
 package() {
-  cd "$srcdir/$pkgname-$pkgver"
+  cd "$srcdir/$pkgname-$pkgver.$pkgvariant"
   make PREFIX=/usr DESTDIR="$pkgdir" install
   install -Dm644 LICENSE "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
   install -Dm644 README "$pkgdir/usr/share/doc/$pkgname/README"
